src/main/java/duke/Duke.java

```diff
@@ -30,28 +30,27 @@ public static void main(String[] args) throws IOException, DukeException {
         /**
          * Array container for user's to do tasks
          */
-        ArrayList<Task> todoList = new ArrayList<Task>();
-        TaskList taskList = new TaskList(todoList);
+        ArrayList<Task> todoLists = new ArrayList<Task>();
+        TaskList taskLists = new TaskList(todoLists);
 
         // initializing storage
         Storage storage = new Storage();
 
         // load data when duke starts up
         try {
-            todoList = storage.load();
-            taskList = new TaskList(todoList);
+            todoLists = storage.load();
+            taskLists = new TaskList(todoLists);
```

Comment:
Great effort changing even variable names to plural to comply with the naming standard!